applications/smart-distancing/ui/utils/visualization_utils.py

```diff
@@ -390,6 +391,38 @@ def visualize_boxes_and_labels_on_image_array(
     return image
 
 
+
+def checkDistance(nn_out):
+
+    # unNormalize centroid of boxes!
+    w,h = 1920,1280
+
+    for obj in nn_out:
+        centroidBox = obj["centroid"]
+        xc = centroidBox[0]*w
+        yc = centroidBox[1]*h
+        wc = centroidBox[2]*w
+        hc = centroidBox[3]*h
+        obj["centroidReal"]=[xc, yc, wc, hc]
```

Comment:
It's better to use normalized numbers (numbers between 0-1). I think work with normalized numbers is more robust. 
@mhejrati how do you think?

Reply:
w,h = 1920,1280 why?
I think you should read it from the original frames.

Reply:
> It's better to use normalized numbers (numbers between 0-1). I think work with normalized numbers is more robust.
> @mhejrati how do you think?

why not robust? I moved content of this loop to one of previous ones that is used to make `bbox` coordinates in core.py and I will send another PR.